mobile/src/components/Button.tsx

```diff
@@ -47,7 +43,6 @@ const themedStyles = (type: ButtonType) =>
       borderRadius: 100,
```

Comment:
Please use withStyles to get the values from the theme and also for the shadow the color should not be free text